sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/nlp/automl_nlp_job.py

```diff
@@ -25,6 +25,38 @@
 
 # pylint: disable=too-many-instance-attributes,protected-access
 class AutoMLNLPJob(AutoMLVertical, ABC):
+    """Base class for AutoML NLP jobs.
+
+    You should not instantiate this class directly. Instead you should
+    create classes for specific NLP Jobs.
+
+    :param task_type: NLP task type, must be one of 'TextClassification',
+        'TextClassificationMultilabel', or 'TextNER'
+    :type task_type: str
+    :param primary_metric: Primary metric to display from NLP job
+    :type primary_metric: str
+    :param training_data: Training data
+    :type training_data: Input
+    :param validation_data: Validation data
+    :type validation_data: Input
+    :param target_column_name: Column name of the target column., defaults to None
+    :type target_column_name: Optional[str], optional
+    :param log_verbosity: The degree of verbosity used in logging, defaults to None,
+        must be one of 'NotSet', 'Debug', 'Info', 'Warning', 'Error', or 'Critical'
+    :type log_verbosity: Optional[str], optional
+    :param featurization: Featurization settings used for NLP job, defaults to None
+    :type featurization: Optional[NlpFeaturizationSettings], optional
+    :param limits: Limit settings for NLP jobs, defaults to None
+    :type limits: Optional[NlpLimitSettings], optional
+    :param sweep: Sweep settings used for NLP job, defaults to None
+    :type sweep: Optional[NlpSweepSettings], optional
```

Comment:
"Optional[NlpSweepSettings], optional" ==> Why optional twice?

Reply:
Yep, should be `Optional[~azure.ai.ml.automl.NlpSweepSettings]`